custom_components/powercalc/data/sonos/connect/model.json

```diff
@@ -0,0 +1,18 @@
+{
+    "measure_description": "Streamed 'Pink noise' from https://www.genelec.com/audio-test-signals and measured using Docker container.",
+    "measure_method": "script",
+    "measure_device": "TP-Link Kasa KP115",
+    "name": "Sonos Connect",
+    "standby_power": 5.0,
+    "device_type": "smart_speaker",
+    "supported_modes": [
+        "fixed"
+    ],
+    "calculation_enabled_condition": "{{ is_state('[[entity]]', 'playing') }}",
+    "fixed_config": {
+        "power": 5.33
+    },
+    "aliases": [
+        "Connect"
```

Comment:
Alias can be removed as it is the same as the directory name.
Directory name will also be checked when auto discovering.